src/FrameBuffer.cpp

```diff
@@ -707,16 +707,24 @@ void FrameBufferList::attachDepthBuffer()
 {
 	if (m_pCurrent == nullptr)
 		return;
+#ifdef VC
+	const GLenum discards[]  = {GL_DEPTH_ATTACHMENT};
+	glDiscardFramebufferEXT(GL_FRAMEBUFFER, 1, discards);
+#endif
```

Comment:
It is only known to benefit the Raspberry Pi, gonetz said this in regards to this line in the other pull request:

> It is wrong functionality. If it fixes issues with some devices, enable it for these devices only.